mir_perception/mir_object_recognition/ros/scripts/pc_object_recognizer_node

```diff
@@ -79,22 +88,39 @@ class PointcloudObjectRecognizer():
         :param object_list:     Object list
         :type:                    mas_perception_msgs.object_list
         """
-        
+
         if object_list.objects:
             rospy.loginfo("%d clouds received ", len(object_list.objects))
             recognized_object_list = ObjectList()
             recognized_object_list = object_list.objects
+
             for object in recognized_object_list:
                 if self.model == "feature_based":
-                    cloud = self.extract_pointcloud(object.views[0].point_cloud, color="hsv")
+                    cloud = self.extract_pointcloud(
+                        object.views[0].point_cloud, color="hsv")
                     features = self.fe_method(cloud)
-                    features = np.reshape(features, (1,-1))
+                    features = np.reshape(features, (1, -1))
                     name, probability = self.classifier.classify(features)
                     name = name[0]
                 elif self.model == "cnn_based":
-                    cloud = self.extract_pointcloud(object.views[0].point_cloud, color="rgb")
-                    label, probability = self.classifier.classify(cloud, center=True, rotate=True, pad=True)
-                    name = self.label_to_name[label]
+                    cloud = self.extract_pointcloud(
+                        object.views[0].point_cloud, color="rgb")
+
+                    label, probability = self.classifier.classify(
+                        cloud, center=True, rotate=True, pad=True)
+
+                    if (label, probability) != (None, None):
+                        name = self.label_to_name[label]
+                        probability = probability
+                    else:
+                        name = "unknown object found"
+                        probability = 0.0
+                        continue
+
+                    print('\n######################')
+                    print('name: ', name)
```

Comment:
is this needed? If yes, replace by rospy.loginfo or logdebug

Reply:
replaced with rospy.loginfo